system/src/system.cpp

```diff
@@ -0,0 +1,98 @@
+#include "lute/system.h"
+#include "lua.h"
+#include "uv.h"
+#include <iterator>
+#include <string>
+#include <vector>
+#include <winnt.h>
+
+int luaopen_system()
+{
+    return 0;
+}
+
+int luteopen_system(lua_State* L)
+{
+    lua_createtable(L, 0, std::size(system_lib::lib));
```

Comment:
This initial size isn’t accurate since we’re adding additional properties below.

Reply:
So that'd be std::size(systemlib::lib) + 3, right

Reply:
Yes, although having something that lists the properties in the header would probably be nice tbh. A uniform structure is nice here, and also makes it easier to look at the headers to see what they’re exposing.